AppGenGuide.md

```diff
@@ -245,48 +235,42 @@ The structure of the app inside the tarball will be as follows
+> At this point, the generated package is a working StarlingX App, however it
+> contains empty templates for some files. The following sections will describe
+> how to further enhance your StarlingX App. 
 
 ## Customizing the application
 
-If you wish to customize the Flux and the plugins for the particularities of
-the application, it is important to modify some of the generated files.
-
-In order to allow such customization, the generator allows the user to decide
-whether to execute the entire generator or only create the files to be modified
-to be packaged later.
+If you wish to customize Flux and the plugins for the particularities of
+your application, it is important to modify some of the generated files.
 
-The entire set of options for calling the generator is
+In order to allow such customization, the generator provides additional
+functions to modify specific files in the package.
 
-`$ python3 app-gen.py -i app_manifest.yaml [-o ./output] [--overwrite] [--no-package] [--package-only]`
+```shell
+python3 app-gen.py -i app_manifest.yaml [-o ./output] [--overwrite] [--no-package]|[--package-only]
+```
 
 Where:
 
-- ``-i/--input`` Input app_manifest.yaml file
-- ``-o/--output`` Output folder, if none is passed the generator will create a folder
-  with the app name in the current directory.
-- ``--overwrite`` Delete existing folder with the same name as the app name
-- ``--no-package`` Only creates the fluxcd manifest, the plugins and the
-  metadata file
-- ``--package-only`` Create the plugins wheels, sha256 file, helm-chart tarball
-  and package the entire application into a tarball.
-
-In order to be able to modify the plugins and overrides before packaging the 
-application the user will need to use `--no-package` in the python call. By using
-this option the generator will:
-
-- Generate the FluxCD Manifest;
-- Generate the plugins with an empty implementation;
-- Generate the metadata.yaml
-  
-After creating the above files, the app generator will end its execution.
+- `-i/--input`: path to the `app_manifest.yaml` configuration file.
+- `-o/--output`: output folder. Defaults to a new folder with the app name in
+the current directory.
+- `--overwrite`: deletes existing output folder before starting.
+- `--no-package`: only creates the FluxCD manifest, plugins and the
+  metadata file, without compressing them in a tarball.
+- `--package-only`: create the plugins wheels, sha256 file, helm-chart tarball
+  and package the entire application into a tarball. TODO: I don't understand how this option goes together with the one above.
+
+This means that, in order to be able to make additional configuration, one must:
+
+- first run the App Generator with `--no-package`;
+- then do the changes (described in the following sections);
+- finally, run the App Generator again with `--package-only`.
 
 ### FluxCD Manifest
 
-[//]: # (TODO: Validate this information)
+[//]: # (TODO: Validate this information - what needs validation?)
```

Comment:
@Danmcaires

Reply:
I believe it was something about the setup.cfg, but I will need some time to remember, sorry.